src/System.Security.Principal.Windows/src/System/Security/Principal/SID.cs

```diff
@@ -50,68 +51,198 @@ internal enum SidNameUse
         AccountComputersSid = 44,
+        /// <summary>Indicates a SID that matches the account controller group.</summary>
         AccountControllersSid = 45,
+        /// <summary>Indicates a SID that matches the certificate administrators group.</summary>
         AccountCertAdminsSid = 46,
+        /// <summary>Indicates a SID that matches the schema administrators group.</summary>
         AccountSchemaAdminsSid = 47,
+        /// <summary>Indicates a SID that matches the enterprise administrators group.</summary>
         AccountEnterpriseAdminsSid = 48,
+        /// <summary>Indicates a SID that matches the policy administrators group.</summary>
         AccountPolicyAdminsSid = 49,
+        /// <summary>Indicates a SID that matches the RAS and IAS server account.</summary>
         AccountRasAndIasServersSid = 50,
+        /// <summary>Indicates a SID present when the Microsoft NTLM authentication package authenticated the client.</summary>
         NtlmAuthenticationSid = 51,
+        /// <summary>Indicates a SID present when the Microsoft Digest authentication package authenticated the client.</summary>
         DigestAuthenticationSid = 52,
+        /// <summary>Indicates a SID present when the Secure Channel (SSL/TLS) authentication package authenticated the client.</summary>
         SChannelAuthenticationSid = 53,
+        /// <summary>Indicates a SID present when the user authenticated from within the forest or across a trust that does not have the selective authentication option enabled. If this SID is present, then <see cref="WinOtherOrganizationSid"/> cannot be present.</summary>
         ThisOrganizationSid = 54,
+        /// <summary>Indicates a SID present when the user authenticated across a forest with the selective authentication option enabled. If this SID is present, then <see cref="WinThisOrganizationSid"/> cannot be present.</summary>
         OtherOrganizationSid = 55,
+        /// <summary>Indicates a SID that allows a user to create incoming forest trusts. It is added to the token of users who are a member of the Incoming Forest Trust Builders built-in group in the root domain of the forest.</summary>
         BuiltinIncomingForestTrustBuildersSid = 56,
+        /// <summary>Indicates a SID that matches the performance monitor user group.</summary>
         BuiltinPerformanceMonitoringUsersSid = 57,
+        /// <summary>Indicates a SID that matches the performance log user group.</summary>
         BuiltinPerformanceLoggingUsersSid = 58,
+        /// <summary>Indicates a SID that matches the Windows Authorization Access group.</summary>
         BuiltinAuthorizationAccessSid = 59,
+        /// <summary>Indicates a SID is present in a server that can issue terminal server licenses.</summary>
         WinBuiltinTerminalServerLicenseServersSid = 60,
-        MaxDefined = WinBuiltinTerminalServerLicenseServersSid,
+        [Obsolete, EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Nit: Could you please split it into 2 lines? It's easier to read + AFAIK it is the general unwritten rule in the code base ...
```c#
[Obsolete]
[EditorBrowsable(EditorBrowsableState.Never)]
```